contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -25,12 +27,30 @@ contract LPOptimiserStrategy is DefaultAccessControl, ILpCallback {
     IVoltzVault internal _vault;
     uint256[] internal _pullExistentials;
     IMarginEngine internal _marginEngine;
+    IPeriphery internal _periphery;
+    IVAMM internal _vamm;
     IVoltzVault.TickRange internal _currentPosition;
 
     // MUTABLE PARAMS
 
     uint256 _lastSignal;
     uint256 _lastLeverage;
+    uint256 _k_unwind_parameter; // parameter for k*leverage (for unwinding so this needs to be sent to the contract vault but not used in the strategy vault)
+    uint256 _sigmaWad; // y (standard deviation parameter in wad 10^18)
+    int256 _max_possible_lower_bound_wad; // should be in fixed rate
+
+    int24 _logProximity; // x (closeness parameter in wad 10^18) in log base 1.0001
+    int24 _currentTick;
+    int24 _tickLower;
+    int24 _tickUpper;
+    int24 _tickSpacing;
```

Comment:
It's good to explicitly mention the modifier for each variable. For example, these should be internal. Moreover, variables as sigma wad, max bound, log proximity should have independent setters and getters.

Reply:
These are all declared as internal now